server/routes/model.routes.js

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import { authenticate, isAdminOrUserOrVendor } from '../helpers/middleware';
+import ModelController from '../controllers/model.controllers';
+
+const router = express.Router();
+
+/**
+ * @swagger
+ * path:
+ *  /:
+ *    get:
+ *      summary: Returns all models in database
```

Comment:
Please fix for description too below:

```suggestion
 *      summary: Count the total number of each model in the database
```